insta-unfollower.py

```diff
@@ -223,10 +232,7 @@ def main():
 
         time.sleep(random.randint(2, 4))
 
-    if len(sys.argv) > 1:
-        connected_user = sys.argv[1]
-    else: 
-        get_user_profile(os.environ.get('INSTA_USERNAME'))
+    connected_user = get_user_profile(credentials.password)
```

Comment:
we finally update the `connected_user` content by once again calling the creds class